openedx/core/lib/xblock_pipeline/finder.py

```diff
@@ -0,0 +1,142 @@
+"""
+Django pipeline finder for handling static assets required by XBlocks.
+"""
+from datetime import datetime
+import os
+from pkg_resources import resource_exists, resource_listdir, resource_isdir, resource_filename
+
+from xblock.core import XBlock
+
+from django.contrib.staticfiles import utils
+from django.contrib.staticfiles.finders import BaseFinder
+from django.contrib.staticfiles.storage import FileSystemStorage
+from django.core.files.storage import Storage
+
+
+class XBlockPackageStorage(Storage):
```

Comment:
Couldn't this derive from FileSystemStorage, and not have to replicate a lot of the code?